### ISSUE TITLE
Random crash when calling ActionSheetDialog

### ISSUE THREAD

**Issue by victorlsn** [devxoul/Toaster#103]
Sometimes when I try to call a ActionSheetDialog on my app, the app crashes.
As I don't know what causes the crash (sometimes it happens, sometimes it doesn't) I can't reproduce it with precision. But the part of the code that is crashing the app is in the Toaster Library, on this part:

  override open var rootViewController: UIViewController? {
    get {
      guard !self.isStatusBarOrientationChanging else { return nil }
      guard let firstWindow = UIApplication.shared.windows.first else { return nil }
      return firstWindow is ToastWindow ? nil : firstWindow.rootViewController
    }
    set { /* Do nothing */ }
  }

More specifically, it's on this line
      guard let firstWindow = UIApplication.shared.windows.first else { return nil }

Do you guys have any idea of what could be causing this? Sorry for being so vague but I really have no idea of how to reproduce it and gather more info.

**Comment by victorlsn** [comment]
I believe this is related to this pull:
https://github.com/devxoul/Toaster/pull/98

**Comment by devxoul** [comment]
@victorlsn, can you try with the code:

```diff
- guard let firstWindow = UIApplication.shared.windows.first else { return nil }
+ guard let firstWindow = UIApplication.shared.delegate?.window ?? nil else { return nil }
```

**Comment by devxoul** [comment]
Fixed in #106
